UM/Resources.py

```diff
@@ -264,7 +264,7 @@ def __find(cls, resource_type, *args):
     @classmethod
     def __initializeStoragePaths(cls):
         if platform.system() == "Windows":
-            cls.__config_storage_path = os.path.join(os.path.expanduser("~\\AppData\\Local\\"), cls.ApplicationIdentifier)
+            cls.__config_storage_path = os.path.join(os.getenv("LOCALAPPDATA"), cls.ApplicationIdentifier)
```

Comment:
`LocalAppData` is introduced in Vista. I wonder if we should check the windows version, and use `os.path.expanduser("~\\AppData\\Local\\")` as a backwards compatible approach for earlier Windows versions?